domains/runtime/auto-id/src/lib.rs

```diff
@@ -938,6 +982,22 @@ impl_runtime_apis! {
             ).map_err(|err| DecodeExtrinsicError(format!("{}", err)))
         }
 
+        fn decode_extrinsics_prefix(
```

Comment:
not sure I understand what prefix means here

Reply:
Please take a look at the comment where this is defined, naming is hard, always open to better suggestions.